tools.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
```

Comment:
I think we don't need this line?

Reply:
This line is automatically generated by `go fmt` (go 1.17).

Should we delete it?

https://tip.golang.org/doc/go1.17#gofmt

Also look forward a few lines from this anchor, tag `//go:build lines`.

Reply:
Got it. I think it's okay. Thank you for your information.